### PR TITLE
Changed all hyperion hk references from float to uint or int. Updated…

### PR DESCRIPTION
**Pull request by nsorense4**
… 2U hyperion hk.

### CODE REVIEW

File: hardware_interface/include/hyperion.h [hunk 1]
```diff
@@ -22,59 +22,63 @@
...
+    uint16_t Port_Voltage;
+    uint16_t Port_Dep_Voltage;
+    uint16_t Star_Voltage;
+    uint16_t Star_Dep_Voltage;
+    uint16_t Zenith_Voltage;
+    uint16_t Port_Current;
+    uint16_t Port_Dep_Current;
+    uint16_t Star_Current;
+    uint16_t Star_Dep_Current;
+    uint16_t Zenith_Current;
```
Comment (Taylorr82) [inline hardware_interface/include/hyperion.h:76]: I wonder what the size difference would be.. I like this change

File: hardware_interface/source/hyperion.c [hunk 2]
```diff
@@ -68,66 +68,76 @@ void hyperion_config_1_value(config_1_panel_t panel, config_1_channel_type_t cha
...
         adc_init(slave_addr, 1 << 7);
         adc_get_raw(slave_addr, &data, &ch); // invalid value
         adc_get_raw(slave_addr, &data, &ch); // correct value
-        *param = adc_calculate_sensor_temp(data, ADC_VREF);
+        int8_t *result = (int8_t *)param;
+        *result = (int8_t)(adc_calculate_sensor_temp(data, ADC_VREF));
         break;
-
-    case CONFIG_1_CHANNEL_TEMP_2:
+    }
```
Comment (yunqingyi) [inline hardware_interface/source/hyperion.c:78]: do we need curly brackets for switch cases?

**Review by yunqingyi — APPROVED** [review]

File: hardware_interface/source/hyperion.c (hunk 2 above)
Comment (Taylorr82) [reply hardware_interface/source/hyperion.c:78]: If you are going to use them there should be a space between the case and the {

File: hardware_interface/source/hyperion.c [hunk 3]
```diff
@@ -68,66 +68,76 @@ void hyperion_config_1_value(config_1_panel_t panel, config_1_channel_type_t cha
 
     switch (channel) {
     /* Channel 1 to 3 handle temp sensor */
-    case CONFIG_1_CHANNEL_TEMP_1:
+    case CONFIG_1_CHANNEL_TEMP_1:{
         adc_init(slave_addr, 1 << 7);
```
Comment (Taylorr82) [inline hardware_interface/source/hyperion.c:72]: What do these numbers mean? Should each of these 1 << <n> be their own macro? I suspect they should

**Review by Taylorr82 — CHANGES_REQUESTED** [review]

File: hardware_interface/source/hyperion.c (hunk 2 above)
Comment (nsorense4) [reply hardware_interface/source/hyperion.c:78]: Done.

File: hardware_interface/source/hyperion.c (hunk 3 above)
Comment (nsorense4) [reply hardware_interface/source/hyperion.c]: I added the macros.

**Review by tganley — APPROVED** [review]

**Review by Taylorr82 — APPROVED** [review]

**Pull request merged**